docker-compose.yml

```diff
@@ -16,6 +16,8 @@ services:
       - ../dump:/docker-entrypoint-initdb.d
       # for importing a dump
       # if you want to re-import simply put away ../mysql/*
+  redis:
+    image: redis:latest
```

Comment:
hi @icarito , I have added redis image here. Does it require anything else?
I have tested it locally using `docker-compose up` and it went well I guess. Please have a look!!
Thanks!!